packages/webview_flutter/webview_flutter_wkwebview/example/lib/web_view.dart

```diff
@@ -227,6 +228,13 @@ class WebView extends StatefulWidget {
   /// The default policy is [AutoMediaPlaybackPolicy.require_user_action_for_all_media_types].
   final AutoMediaPlaybackPolicy initialMediaPlaybackPolicy;
 
+  /// The background color of the webview.
+  ///
+  /// When null the platform's webview default background color is used.
+  ///
+  /// By default `backgroundColor` is null.
```

Comment:
nit: Could be combined into one paragraph:
```suggestion
  /// When `null` the platform's webview default background color is used. By 
  /// default [backgroundColor] is `null`.
```